### PR TITLE
email templates for UG sponsorship approval

### PR DESCRIPTION
**Pull request by carmalou**

### CODE REVIEW

**Comment by kacollins** [comment]
Typo in the two acceptance templates: `each of groups reaches`

**Comment by kacollins** [comment]
Changes to the accepted/sponsored template:
Head count is not required.
Photo of the meetup in action should be posted on meetup.com, not in the GitHub issue comment.
Add the requirement that the meetup description must be accurate and informative.
Should this also have a link to the StarSpace46 User Group Leader Agreement?

File: email-templates/accepted-sponsored.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Your group is important to the Oklahoma tech community, and Techlahoma is excited to sponsor your group! Please note that we use [Meetup](link) for events. Meetup allows us to carefully track the number of Oklahomans each of groups reaches, which in turn helps us know where to focus our efforts.
+
+{ if approved for SS46 }
+
+Additionally, we are pleased to offer space for your group to meet at [StarSpace46](link) at { dateAndTime }. Your event space reservation cannot be completed until your event is scheduled on Meetup, so _please schedule your events promptly._
+
+{ if approved for food/drink reimbursement }
+
+We know it's easier to provide the content your group provides when you don't have to worry about food and drinks. We have allotted funds to reimburse your group for food, up to $4 per attendee.
```
Comment (seejessicacode) [inline email-templates/accepted-sponsored.md]: $4 per Meetup event RSVP instead of attendee

File: email-templates/accepted-sponsored.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Dear { usergroup_name },
+
+We are pleased to inform you that the Techlahoma Board approved { usergroup_name } for { year }.
+
+Your group is important to the Oklahoma tech community, and Techlahoma is excited to sponsor your group! Please note that we use [Meetup](link) for events. Meetup allows us to carefully track the number of Oklahomans each of groups reaches, which in turn helps us know where to focus our efforts.
+
+{ if approved for SS46 }
+
+Additionally, we are pleased to offer space for your group to meet at [StarSpace46](link) at { dateAndTime }. Your event space reservation cannot be completed until your event is scheduled on Meetup, so _please schedule your events promptly._
```
Comment (seejessicacode) [inline email-templates/accepted-sponsored.md]: Add reference to the StarSpace46 User Group Leader Agreement

Comment (seejessicacode) [reply email-templates/accepted-sponsored.md]: Could also include instructions to notify Techlahoma of any date changes or canceled events with no less than 36 hours notice so we can update those reservations. SS46's cobot reservation tool doesn't allow any changes up 24 hours before the reservation.

File: email-templates/accepted-sponsored.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+{ if approved for food/drink reimbursement }
+
+We know it's easier to provide the content your group provides when you don't have to worry about food and drinks. We have allotted funds to reimburse your group for food, up to $4 per attendee.
+
+In order to arrange your reimbursement, please provide the following:
+
+- A picture of the receipt
+- A final headcount for the event (including organizers)
```
Comment (seejessicacode) [inline email-templates/accepted-sponsored.md]: Headcount is no longer necessary

File: email-templates/accepted-sponsored.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- A photo of the meetup in action
```
Comment (seejessicacode) [inline email-templates/accepted-sponsored.md]: Posted to the meetup event

File: email-templates/accepted-sponsored.md (hunk 2 above)
Comment (groovecoder) [reply email-templates/accepted-sponsored.md]: Can we make sure this text is only sent to OKC groups?

Comment (groovecoder) [reply email-templates/accepted-sponsored.md]: Oops, now I see the `if approved for SS46` check.

@dustincurzon, what do you think abou this?

```
{ if approved for 36N }

Additionally, we are pleased to offer space for your group to meet at [36°N](link) at { dateAndTime }. Please contact Sarah McKelvey to confirm your event space.
```

**Comment by dustincurzon** [comment]
I think that makes sense.

File: email-templates/accepted-sponsored.md (hunk 2 above)
Comment (groovecoder) [reply email-templates/accepted-sponsored.md]: That sounds right. So, maybe drop the `at { dateTime}` part. We can offer it, but they'll have to coordinate the dates & times with Sarah @ 36°N.

Comment (seejessicacode) [reply email-templates/accepted-sponsored.md]: I'll need contact info but you don't have to share it here.

**Comment by carmalou** [comment]
Ok, so I think I got everything. Fixed typo, added 36 degrees north note, removed head count requirement && clarified food/drink reimbursement.

**Comment by seejessicacode** [comment]
This looks great! Thank you for doing this Carmen!

**Pull request merged**